e2e/packages/server-extensibility/types.ts

```diff
@@ -18,6 +18,20 @@ interface ServerExtensibility extends Package {
      */
     serverExtensibility: React.ElementType;
   };
+
+  /**
+   * Server middleware.
+   */
+  server?: {
+    /**
+     * ServerExtensibility namespace.
+     */
+    serverExtensibility: {
+      [key: string]:
```

Comment:
Would you mind using the specific names (`robotsTxtTwo`, `accessFrontityState`...) of the functions here? This is going to be the best example of how to type these functions in our codebase.

Reply:
Oki doki!